### ISSUE TITLE
Window panes not adjustable

### ISSUE THREAD

**Issue by sse450** [r-pufky/digikam#6]
digikam has basically 3 vertical panes. Middle one is for thumbnails and previews. Normally, vertical bars are horizontally adjustable to make more room to the middle space. But, the panes are not adjustable.

![2020-10-17_11-12](https://user-images.githubusercontent.com/15627450/96331928-d66cd800-1069-11eb-88c4-ed88b1f35bea.png)

**Comment by r-pufky** [comment]
Double check your grab area. I've just tested this is in my containers and it works fine.

![original](https://user-images.githubusercontent.com/4778046/96351783-8fcfab80-1072-11eb-8c39-9fb637e4796c.png)

Then moved:
![mod](https://user-images.githubusercontent.com/4778046/96351784-94945f80-1072-11eb-8ac9-f87301d281bd.png)

I've confirmed this works with sub windows too.

What might be occurring is click targets shifting due to resolution scaling, etc. Try grabbing closer to one edge or another and see if that works.
